### PR TITLE
Added Code of Conduct and Contributing guides

### PR DESCRIPTION
**Pull request by ElliotThiebaut**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+## Contributing by creating pull requests
+
+If you want to directly contribute to the project by adding or fixing some code, you can create a pull request with your desired changes. Here are some guidelines to follow:
+
+- Fork the repository into your own account.
+
+- Checkout a topic branch from `main` with a name like `feature/adding-types` or `bug/docs-images-error`.
+
+- Make sure to run `npm run lint` and `npm run prettier` before committing your changes. You can even enable GitHub actions in your forked repository to run these checks automatically.
```
Comment (charliefoxtwo) [inline CONTRIBUTING.md:19]: Should this be lint:fix and prettier:fix?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+- Create a pull request with your changes from your feature branch against the `main` branch.
+
+- If adding a new feature:
+
+  - Provide a convincing reason to add this feature. Ideally, you should open a suggestion issue first and have it approved before working on it.
+
+- If fixing bug:
```
Comment (charliefoxtwo) [inline CONTRIBUTING.md:27]:
```suggestion
- If fixing a bug:
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+  - Provide a detailed description of the bug in the PR. Live demo preferred.
+
+- It's OK to have multiple small commits as you work on the PR - GitHub can automatically squash them before merging.
```
Comment (charliefoxtwo) [inline CONTRIBUTING.md:31]: I would just say something like "all PRs will be squash merged" or something. It's really github-specific git behavior so much as git behavior we force github to follow

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+### Working on the api-client package
+
+You can now start working on the project. To test if everything works correctly, you can build the package:
+
+```sh
+# build the package and the documentation website
+$ npm run build
+```
+
+To test if your changes to the package work correctly, you can use the `npm link` feature and link the package to a test project.
```
Comment (charliefoxtwo) [inline CONTRIBUTING.md:55]: Should we link out to documentation on npm link?

File: CONTRIBUTING.md (hunk 1 above)
Comment (ElliotThiebaut) [reply CONTRIBUTING.md:19]: Yeah true

File: CONTRIBUTING.md (hunk 4 above)
Comment (ElliotThiebaut) [reply CONTRIBUTING.md]: Added !

File: CONTRIBUTING.md (hunk 3 above)
Comment (ElliotThiebaut) [reply CONTRIBUTING.md]: Changed !

File: CONTRIBUTING.md (hunk 2 above)
Comment (ElliotThiebaut) [reply CONTRIBUTING.md]: Changed !

File: CONTRIBUTING.md (hunk 1 above)
Comment (ElliotThiebaut) [reply CONTRIBUTING.md]: Changed !

**Review by charliefoxtwo — APPROVED** [review]

**Pull request merged**